deploy/gke-marketplace-app/server-deployer/chart/triton/values.yaml

```diff
@@ -31,7 +31,7 @@ maxReplicaCount: 3
 tritonProtocol: HTTP
 # HPA GPU utilization autoscaling target
 HPATargetAverageValue: 85
-modelRepositoryPath: gs://triton_sample_models/23_04
+modelRepositoryPath: gs://triton_sample_models/23_06
```

Comment:
For the future, not sure if it makes sense to parameterize these release versions so that we don't have to update this doc on every release. Not blocking this PR for this though.

Reply:
I think it is parameterized, and automatically updated for each release, is it not?

Reply:
going forward when Triton support the compatiblity feature of TRT, this can be dropped.

Reply:
>I think it is parameterized, and automatically updated for each release, is it not?

Doesn't the fact that this PR exists mean it isn't parameterized? I believe I am missing some understanding here. 

>going forward when Triton support the compatiblity feature of TRT, this can be dropped

That's great news. cc: @tanmayv25 for vis

Reply:
any other concerns? We have an internal jira track this feature. I think there is script to refresh the version already